packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -279,6 +293,18 @@ export abstract class QueueProcessingServiceBase extends Construct {
    * The AwsLogDriver to use for logging if logging is enabled.
    */
   public readonly logDriver?: LogDriver;
+  /**
+   * Flag to disable CPU based auto scaling strategy on the service.
+   *
+   * @default - false
+   */
+  public readonly disableCpuBasedScaling: boolean = false
+  /**
+   * The target CPU utilization percentage for CPU based scaling strategy when enabled.
+   *
+   * @default - 50
+   */
+  public readonly targetUtilizationPercent: number = 50
```

Comment:
```suggestion
  public readonly logDriver?: LogDriver;

  /**
   * Flag to disable CPU based auto scaling strategy on the service.
   */
  public readonly disableCpuBasedScaling: boolean;

  /**
   * The target CPU utilization percentage for CPU based scaling strategy when enabled.
   */
  public readonly targetUtilizationPercent: number;
```
Defaults are assigned in the constructor.

Reply:
Thanks, updated.